specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/examples/CreateAVmWithNetworkInterfaceConfiguration.json

```diff
@@ -0,0 +1,174 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "vmName": "myVM",
+    "api-version": "2021-03-01",
+    "parameters": {
+      "location": "westus",
+      "properties": {
+        "hardwareProfile": {
+          "vmSize": "Standard_D1_v2"
+        },
+        "storageProfile": {
+          "imageReference": {
+            "sku": "2016-Datacenter",
+            "publisher": "MicrosoftWindowsServer",
+            "version": "latest",
+            "offer": "WindowsServer"
+          },
+          "osDisk": {
+            "caching": "ReadWrite",
+            "managedDisk": {
+              "storageAccountType": "Standard_LRS"
+            },
+            "name": "myVMosdisk",
+            "createOption": "FromImage"
+          }
+        },
+        "networkProfile": {
+          "networkApiVersion": "2020-11-01",
+          "networkInterfaceConfigurations": [
+            {
+              "name": "{nic-config-name}",
+              "properties": {
+                "primary": true,
+                "deleteOption": "Delete",
+                "ipConfigurations": [
+                  {
+                    "name": "{ip-config-name}",
+                    "properties": {
+                      "primary": true,
+                      "publicIPAddressConfiguration": {
+                        "name": "{publicIP-config-name}",
+                        "sku": {
+                          "publicIPAddressSkuName": "Basic",
+                          "publicIPAddressSkuTier": "Global"
+                        },
+                        "properties": {
+                          "deleteOption": "Detach",
+                          "publicIPAllocationMethod": "Static"
+                        }
+                      }
+                    }
+                  }
+                ]
+              }
+            }
+          ]
+        },
+        "osProfile": {
+          "adminUsername": "{your-username}",
+          "computerName": "myVM",
+          "adminPassword": "{your-password}"
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/{subscription-id}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/virtualMachines/myVM",
+        "type": "Microsoft.Compute/virtualMachines",
+        "properties": {
+          "osProfile": {
+            "adminUsername": "{your-username}",
+            "secrets": [],
+            "computerName": "myVM",
+            "windowsConfiguration": {
+              "provisionVMAgent": true,
+              "enableAutomaticUpdates": true
+            }
+          },
+          "networkProfile": {
```

Comment:
>networkProfile [](start = 11, length = 14)

Why is the network profile completely different between the PUT request and PUT response?